### PR TITLE
Add audit-rules-must-be-info meta rule

### PR DESCRIPTION
**Pull request by underyx**

### CODE REVIEW

File: javascript/express/security/audit/express-cookie-settings.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ rules:
       Don’t use the default session cookie name
       Using the default session cookie name can open your app to attacks.
       The security issue posed is similar to X-Powered-By: a potential attacker can use it to fingerprint the server and target attacks accordingly.
-    severity: WARNING
+    severity: INFONG
```
Comment (underyx) [inline javascript/express/security/audit/express-cookie-settings.yaml:7]: infong?

Comment (underyx) [reply javascript/express/security/audit/express-cookie-settings.yaml:7]: image

**Comment by colleend** [comment]
I see that there's discussion about whether we should change all audit rules to INFO, and the consensus among security researchers is that we should hold off on this. Should I close this PR @underyx @p4p3r?

**Comment by underyx** [comment]
yup

**Pull request closed without merging**